packages/protocol/contracts/layer1/preconf/impl/LookaheadStore.sol

```diff
@@ -167,98 +179,58 @@ contract LookaheadStore is ILookaheadStore, EssentialContract {
     )
         internal
         view
+        returns (LookaheadPayload[] memory)
     {
-        // Validate the lookahead poster's operator status within the URC
-        IRegistry.OperatorData memory operatorData = urc.getOperatorData(_registrationRoot);
-        require(operatorData.unregisteredAt == 0, PosterHasUnregistered());
-        require(operatorData.slashedAt == 0, PosterHasBeenSlashed());
-        require(
-            operatorData.collateralWei >= getConfig().minCollateralForPosting,
-            PosterHasInsufficientCollateral()
-        );
+        require(_signedCommitment.commitment.slasher == guardian, SlasherIsNotGuardian());
 
-        // Validate the slashing commitment of the lookahead poster
-        IRegistry.SlasherCommitment memory slasherCommitment =
-            urc.getSlasherCommitment(_registrationRoot, guardian);
-        require(slasherCommitment.optedOutAt < slasherCommitment.optedInAt, PosterHasNotOptedIn());
+        (, IRegistry.SlasherCommitment memory slasherCommitment) = _validateOperator(
+            _registrationRoot, block.timestamp, getConfig().minCollateralForPosting, guardian
```

Comment:
note that block.timestamp is used here for calling `getHistoricalCollateral` inside _validateOperator.